src/main/java/io/cryostat/recordings/RecordingArchiveHelper.java

```diff
@@ -213,11 +213,13 @@ public Future<ArchivedRecordingInfo> deleteRecording(String recordingName) {
                             webServerProvider.get().getArchivedReportURL(filename),
                             recordingMetadataManager.deleteRecordingMetadataIfExists(
                                     ARCHIVES, recordingName));
+            byte[] decodedParentPath = base32.decode(parentPath.getFileName().toString());
+            String decodedTargetId = new String(decodedParentPath, StandardCharsets.UTF_8);
```

Comment:
I just realized this doesn't account for the case where `parentPath.getFileName().toString() == "unlabelled"` (i.e. the recording to be deleted is in the uploads subdirectory). We should skip decoding in that case. 

Looks good otherwise.